csharp/ql/src/semmle/code/csharp/metrics/Coupling.qll

```diff
@@ -222,6 +222,10 @@ predicate shareFieldOrProperty(ValueOrRefType t, Method m1, Method m2) {
   )
 }
 
+/**
+ * Holds if `Method` `m` is declared in `ValueOrRefType` `t`, and if there's a
+ * `FieldAccess` or `PropertyAccess` in `m` that accesses `Declaration` `d`.
+ */
```

Comment:
I would say

Holds if the declaring type of method `m` is `t` and `m` accesses declaration `d`, which is either a field or a property.